pkgs/by-name/af/affine/package.nix

```diff
@@ -63,13 +75,31 @@ stdenv.mkDerivation (finalAttrs: {
       let
         supportedArchitectures = builtins.toJSON {
           os = [
+            "aix"
+            "android"
             "darwin"
+            "freebsd"
             "linux"
+            "netbsd"
+            "openbsd"
+            "sunos"
+            "win32"
           ];
           cpu = [
-            "x64"
-            "ia32"
+            "arm"
             "arm64"
+            "ia32"
+            "loong64"
+            "mips"
+            "mipsel"
+            "mips64el"
+            "powerpc64le"
+            "ppc64"
+            "riscv64"
+            "s390"
+            "s390x"
+            "wasm32"
+            "x64"
```

Comment:
Can we keep this to a list of what we realistically need? At least half the platforms are not truly supported.

Reply:
My main concern is that it will be slightly bigger because we download more variants of binaries. We probably onlx need aarch and x86 linux and darwin but only a minor thing to think about.